Bluepill-cli/Bluepill-cli/Simulator/SimulatorMonitor.m

```diff
@@ -121,9 +122,32 @@ - (void)onTestCasePassedWithName:(NSString *)testName inClass:(NSString *)testCl
 - (void)onTestCaseFailedWithName:(NSString *)testName inClass:(NSString *)testClass
                           inFile:(NSString *)filePath onLineNumber:(NSUInteger)lineNumber wasException:(BOOL)wasException {
     NSDate *currentTime = [NSDate date];
-    [BPUtils printInfo:FAILED withString:@"%10.6fs %@/%@",
-                                          [currentTime timeIntervalSinceDate:self.lastTestCaseStartDate],
-                                          testClass, testName];
+
+    NSString *fullTestName = [NSString stringWithFormat:@"%@/%@", testClass, testName];
+
+    BOOL additionalFailure = NO;
+    for (NSString *fullName in self.failedTestCases) {
+        if([fullTestName isEqualToString:fullName]) {
```

Comment:
containsObject: on NSArray does isEqual: instead of isEqualToString: so is not necessarily safe for use here.